From 84cf3230a9680aac3b73c410c2b758760b6d3066 Mon Sep 17 00:00:00 2001
From: Michael Lippautz <mlippautz@chromium.org>
Date: Thu, 27 Jan 2022 14:14:11 +0100
Subject: [PATCH] cppgc: Fix include

Add <utility> to cover for std::exchange.

Bug: v8:12585
Change-Id: Ida65144e93e466be8914527d0e646f348c136bcb
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/3420309
Auto-Submit: Michael Lippautz <mlippautz@chromium.org>
Reviewed-by: Omer Katz <omerkatz@chromium.org>
Commit-Queue: Michael Lippautz <mlippautz@chromium.org>
Cr-Commit-Position: refs/heads/main@{#78820}
---

diff --git a/src/heap/cppgc/prefinalizer-handler.h b/src/heap/cppgc/prefinalizer-handler.h
index bc17c99..c82c91f 100644
--- a/src/heap/cppgc/prefinalizer-handler.h
+++ b/src/heap/cppgc/prefinalizer-handler.h
@@ -5,6 +5,7 @@
 #ifndef V8_HEAP_CPPGC_PREFINALIZER_HANDLER_H_
 #define V8_HEAP_CPPGC_PREFINALIZER_HANDLER_H_
 
+#include <utility>
 #include <vector>
 
 #include "include/cppgc/prefinalizer.h"
